wp-parsely.php

```diff
@@ -1,14 +1,16 @@
 <?php
-/*
+/**
```

Comment:
Is this intentional or for linting reasons? I wonder if it's what's causing the weird github highlights